src/main/java/com/nike/cerberus/store/ConfigStore.java

```diff
@@ -739,4 +739,52 @@ public void setAuditLoggingEnabled(boolean auditLoggingEnabled) {
     public boolean isAuditLoggingEnabled() {
         return getEnvironmentData().isAuditLoggingEnabled();
     }
+
+    /***
+     * Returns whether configs (environment.json, cms/environment.properties) are the same in the regions defined in
+     * environment.json
+     * @return False if the configs are different and true if the configs are the same
+     */
+    public boolean isConfigSynchronized(){
+        EnvironmentData environmentData = getDecryptedEnvironmentData();
+        Map<Regions, RegionData> map = environmentData.getRegionData();
+        String lastProperties = null;
+        String lastEnvironmentData = null;
+        Regions lastRegions = null;
+        boolean result = true;
+        for (Map.Entry<Regions, RegionData> entry: map.entrySet()){
+            Regions currentRegions = entry.getKey();
```

Comment:
nitpick: this is really one "currentRegion", even though the class type is named plural as "Regions"